sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -15531,7 +15595,7 @@
           "encrypted": false,
           "redrawOn": "data",
           "customDefaultValue": "",
-          "calculateValue": "if (data.hasDependents === 'no' && (data.relationshipStatus === 'single') && data.outOfHighSchoolFor4Years === 'no' && data.howWillYouBeAttendingTheProgram === \"Full Time\") {\n  instance.setValue (true);\n}\nelse {\n  instance.setValue (false);\n}",
+          "calculateValue": "if (\r\n  data.hasDependents === \"no\" &&\r\n  data.relationshipStatus === \"single\" &&\r\n  data.outOfHighSchoolFor4Years === \"no\" &&\r\n  data.howWillYouBeAttendingTheProgram === \"Full Time\" &&\r\n  !(data.everInCustodyOfChildWelfare === \"yes\")\r\n) {\r\n  instance.setValue(true);\r\n} else {\r\n  instance.setValue(false);\r\n}",
```

Comment:
Just pasting the code below to make the discussion easier 😄 
Is there any particular reason to not use the "no" in the condition, for instace.
```js
data.everInCustodyOfChildWelfare === "no"
```

```js
if (
  data.hasDependents === "no" &&
  data.relationshipStatus === "single" &&
  data.outOfHighSchoolFor4Years === "no" &&
  data.howWillYouBeAttendingTheProgram === "Full Time" &&
  !(data.everInCustodyOfChildWelfare === "yes")
) {
  instance.setValue(true);
} else {
  instance.setValue(false);
}
```